sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/COESummaryData.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <v-row class="m-2">
+    <v-col class="category-header-medium color-blue">{{ header }}</v-col>
+  </v-row>
+  <div class="mx-5 py-4">
+    <content-group>
+      <DataTable
+        :autoLayout="true"
+        :value="applications"
+        class="p-m-4"
+        :paginator="true"
+        :rows="10"
```

Comment:
we do have a constants called `DEFAULT_PAGE_LIMIT`

Reply:
As mentioned in PR description, Pagination, Sort and Search will be added in next part of PR.